packages/mux/src/helpers.ts

```diff
@@ -0,0 +1,71 @@
+import { type FilterOperator } from '@rabbitholegg/questdk'
+import {
+  getReaderContract,
+  getChainStorage,
+  type Asset,
+} from '@mux-network/mux.js'
+import { CHAIN_ID_TO_PROVIDER } from './provider'
+import { type Address, zeroAddress, getAddress } from 'viem'
+import { Chains } from './utils'
+
+export const chainToWeth: { [chainId: number]: Address } = {
+  [Chains.ARBITRUM_ONE]: '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1',
+  [Chains.OPTIMISM]: '0x4200000000000000000000000000000000000006',
+}
+
+const assetsCache = new Map()
+
+export const getMuxTokenId = async (
+  chainId: number,
+  tokenAddress: Address,
+): Promise<string | undefined> => {
+  if (!tokenAddress) return undefined
+  const provider = CHAIN_ID_TO_PROVIDER[chainId]
+  if (!provider) {
+    throw new Error(`No provider found for chain ${chainId}`)
+  }
+
+  // cache assets for each chain to reduce api calls
```

Comment:
This won't persist - makes sense to cache the call but we'd need a way to persist the cache across runs. We only run plugins once per quest creation though so the frequency is so low it shouldn't matter.